extensions/linz/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+[unreleased]: https://github.com/stac-extensions/template/compare/v1.0.0...HEAD
```

Comment:
I believe this link should point to this repo.

Reply:
Done